scripts/ecs-anywhere-install.sh

```diff
@@ -592,6 +602,73 @@ wait-agent-start() {
     fail
 }
 
+exec-setup() {
+    find-copy-certs-exec
+    download-ssm-binaries-exec
+}
+
+find-copy-certs-exec() {
+    # Reference for locations: https://golang.org/src/crypto/x509/root_linux.go
+    CERTS_FILES_LOCATIONS=("/etc/ssl/certs/ca-certificates.crt"
+      "/etc/pki/tls/certs/ca-bundle.crt"
+      "/etc/ssl/ca-bundle.pem"
+      "/etc/pki/tls/cacert.pem"
+      "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+      "/etc/ssl/cert.pem"
+    )
+    CERTS_PATH="/var/lib/ecs/deps/execute-command/certs"
+
+    echo "Copying certs for exec feature"
+
+    # Determine certs file
+    certs=""
+    if [ -n "$CERTS_FILE" ] && [ -f "$CERTS_FILE" ]; then
+        certs=$CERTS_FILE
+    else
+        if [ -n "$CERTS_FILE" ]; then
+            echo "Provided certs file does not exist, looking for certs in known possible locations"
+        fi
+        for f in "${CERTS_FILES_LOCATIONS[@]}"
+        do
+            if  [ -f "$f" ]; then
+                certs=$f
+                break
+            fi
+        done
+    fi
+
+    # Copy certs to exec directory
+    if [ -z "$certs" ]; then
+        echo "Could not find certificates. Please rerun with --certs-file and provide a valid path"
+        fail
+    else
+        echo "Using $certs"
+        mkdir -p $CERTS_PATH && cd $CERTS_PATH
+        cp "$certs" tls-ca-bundle.pem
+        chmod 400 tls-ca-bundle.pem
+    fi
+    ok
+}
+
+download-ssm-binaries-exec() {
+    BINARY_VERSION="3.1.338.0"
```

Comment:
Is there a specific reason we use this version? Is there a `latest` that we can use?

Reply:
There will be a stable version endpoint available 12/08. We're hardcoding version for now to access what's currently available, but around that date I'll update the script in this branch to use that

Reply:
we're currently using 3.1.90.0 in the ecs ami: https://github.com/aws/amazon-ecs-ami/blob/main/variables.pkr.hcl#L59

we should probably use the same version, either by downgrading this version or following up with an ecs ami PR to update the version the AMI uses.

If we do update, might be worth bumping to SSM's latest available version which is 3.1.501.0: https://github.com/aws/amazon-ssm-agent/releases

Reply:
Thanks for pointing that out, it looks like there are a couple places where this might be inconsistent. For now at least I'll update to `3.1.501.0` in this script, though before releasing this we will be updating to a url that just points to the latest version. As part of that, I can look into how we can get that elsewhere and remove some of the hardcoded versions